release-notes/opensearch-security-dashboards-plugin.release-notes-2.8.0.0.md

```diff
@@ -5,3 +5,8 @@ Compatible with Opensearch-Dashboards 2.8.0
 ### Bug Fixes
 
 * Fix the test cases to adapt the security password validation feature ([#1437](https://github.com/opensearch-project/security-dashboards-plugin/pull/1437))
+
+### Enhancements
+
+* Add new cluster permissions constants for long-running operation notification feature in Index-Management repo ([#1444](https://github.com/opensearch-project/security-dashboards-plugin/pull/1444))
```

Comment:
nit: No need for the word `new` in release notes in either of these entries. Its implied that these are new additions given that they are going into the release.